app/assets/javascripts/discourse/app/templates/tag-groups.hbs

```diff
@@ -1,3 +1,5 @@
+<a href="/tags">{{d-icon "chevron-left"}}<span>{{i18n "tagging.groups.back_btn"}}</span></a>
```

Comment:
<img width="1259" alt="image" src="https://user-images.githubusercontent.com/101828855/167643269-01f9ab6d-76ee-419f-ab88-c5ffdb6fddb1.png">

I don't know what it wants from me… 🙁

Reply:
```suggestion
<a href="/tags">
  {{d-icon "chevron-left"}}
  <span>{{i18n "tagging.groups.back_btn"}}</span>
</a>
```

Reply:
Also, the PR title suggests a button, but it's implemented as a link? Should it be a `{{d-button}}` instead?

Reply:
It wants `{{d-icon...` to be on a new line. 

One more thing I noticed here, we can't use `/tags` directly, because some Discourse instances are in a subfolder. We could use the `link-to` helper though, something like: 

```
{{#link-to "tags"}} ...icon + label... {{/link-to}}
```